tests/cases/closedLoopHeart_singleVessel.json

```diff
@@ -1,23 +1,21 @@
 {
     "description": {
-            "description of test case" : "Closed-loop circulation with one vessel (aorta) connected on either side to the heart model."
+        "description of test case": "Closed-loop circulation with one vessel (aorta) connected on either side to the heart model."
     },
     "simulation_parameters": {
-        "number_of_cardiac_cycles": 1,
+        "number_of_cardiac_cycles": 10,
         "number_of_time_pts_per_cardiac_cycle": 10000,
+        "absolute_tolerance": 1e-8,
         "steady_initial": false
     },
     "boundary_conditions": [
         {
             "bc_name": "RCR_aorta",
             "bc_type": "ClosedLoopRCR",
             "bc_values": {
-                "_comment_": "R_total = 1.570879*0.948914 = 1.490629075",
```

Comment:
Comments got removed. This happened in all test cases that were read/written by Python during your edits with multiple `_comment_` keys. The edits will only have one `_comment_` key. As a fix, all comments could be combined into one.